e2e-tests/cypress/tests/integration/channels/channel_settings/more_unreads_position_with_scroll_spec.ts

```diff
@@ -52,48 +52,41 @@ describe('Channel settings', () => {
         cy.apiLogin(mainUser);
         cy.visit(`/${myTeam.name}/channels/off-topic`);
 
-        // # Post message as the second user, in a channel near the top of the list
-        cy.apiGetChannelByName(myTeam.name, channelNames[firstChannelIndex]).then(({channel}) => {
-            cy.postMessageAs({
-                sender: otherUser,
-                message: 'Bleep bloop I am a robot',
-                channelId: channel.id,
-            });
+        // # Wait for channels to load
+        cy.get(`#sidebarItem_${channelNames[firstChannelIndex]}`).should('be.visible');
 
-            // # Scroll down in channels list until last created channel is visible
-            cy.get(`#sidebarItem_${channelNames[lastChannelIndex]}`).scrollIntoView({duration: TIMEOUTS.TWO_SEC});
-            cy.get('.scrollbar--view').scrollTo('bottom');
-        });
+        // # Nudge the scrollbar slightly to make the "More Unreads" pills appear
+        // @hmhealey - They seem to appear automatically on a regular browser, but not when running under Cypress
+        cy.get('#SidebarContainer .simplebar-content-wrapper').scrollTo(0, 1);
 
-        // * After scrolling is complete, "More Unreads" pill should be visible at the top of the channels list
-        cy.get('#unreadIndicatorBottom').should('not.be.visible');
+        // * The bottom "More Unreads" pill should be visible and the top one should not
+        cy.get('#unreadIndicatorTop').should('not.be.visible');
+        cy.get('#unreadIndicatorBottom').should('be.visible');
 
-        // * "More Unreads" pill should be visible at the top of the channels list
-        // # Click on "More Unreads" pill
-        cy.get('#unreadIndicatorTop').should('be.visible').click();
+        // # Click on the "More Unreads" pill to scroll down to the bottom
+        cy.get('#unreadIndicatorBottom').click();
 
-        // # Post as another user in a channel near the bottom of the list, scroll channels list to view it (should be in bold)
-        cy.apiGetChannelByName(myTeam.name, channelNames[lastChannelIndex]).then(({channel}) => {
-            cy.postMessageAs({
-                sender: otherUser,
-                message: 'Bleep bloop I am a robot',
-                channelId: channel.id,
-            });
+        // * The list should have scrolled down to the bottom
+        cy.get(`#sidebarItem_${channelNames[firstChannelIndex]}`).should('not.be.visible');
+        cy.get(`#sidebarItem_${channelNames[lastChannelIndex]}`).should('be.visible');
 
-            // # Scroll down in channels list until last created channel is visible
-            cy.get(`#sidebarItem_${channelNames[firstChannelIndex]}`).scrollIntoView({duration: TIMEOUTS.TWO_SEC});
-            cy.get('.scrollbar--view').scrollTo('top');
-        });
```

Comment:
I ended up rewriting this test since it seems like the original test was a bit flaky and then got weirder as people attempted to fix the flakiness. For example, line 73 clicks "More Unreads" to scroll to the top, and then line 84 scrolls the first channel into view, and then line 85 scrolls to the top, but the comment on line 83 says we're scrolling down to the last channel 🫤